src/ex3/mondayu-logger-todoapp.js

```diff
@@ -0,0 +1,82 @@
+import fs from "fs";
+
+const args = process.argv;
+const currentDirectory = process.cwd() + "\\";
+
+if (fs.existsSync(currentDirectory + "todo.txt") === false) {
```

Comment:
this short hand convention is better
```suggestion
if (!fs.existsSync(currentDirectory + "todo.txt")) {
```